contracts/controller/Avatar.sol

```diff
@@ -17,7 +17,7 @@ contract Avatar is Ownable {
     DAOToken public nativeToken;
     Reputation public nativeReputation;
 
-    event GenericCall(address indexed _contract, bytes _params, bool _success);
+    event GenericCall(address indexed _contract, bytes _params, bool _success, uint _value);
```

Comment:
I think the order of the parameters in the event should be different, and params should also be renamed to _data:
`address indexed _contract, bytes _data, uint _value, bool _success`

Reply:
see 
af108f1